must-gather/collection-scripts/gather_crds

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+source pwait
+max_parallelism=10
+
+# Cluster passed in from main gather
+clusterID=$1
```

Comment:
Do some check if the expected cluster that is passed here is valid, but at least checking if variable is set.

Reply:
this is a copy from `must-gather/collection-scripts/gather_crs`, which has not been modified in 3 years. I do think refactoring code is important here, but would do in a separate PR (and change bash to Go), since its not the goal of this PR

Reply:
That's fine, however you are making few refactoring changes already so ones which are not changing logic, but adding safety checks + readability changes are not harmful here.